src/components/DetailValue.vue

```diff
@@ -70,8 +89,20 @@ export default {
       }
       if (update) {
         e.target.innerText = value;
-        this.$emit("change", value);
+        this.sentNewValue(e.target.innerText);
+      }
+    },
+    sentNewValue(value) {
+      let newValue;
+      if (value.match(/[0-9.]+/)) {
+        newValue = parseFloat(value, 10);
+      } else if (
+        ["true", "false", "null"].indexOf(value.toLowerCase()) !== -1
```

Comment:
what about strings, like "name", or "defaultCursor" like "pointer"?

Reply:
there I used old code for changes...
I will recheck

Reply:
master version also have this problem (